selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/dependencies/OpenExternalLibraryFileAfterRefreshTest.java

```diff
@@ -86,6 +77,6 @@ private void openFileFromExternalLibraryAndCheckDisplayedText() {
 
     editor.waitActive();
     editor.waitTabIsPresent(CHECKING_FILE_NAME);
-    editor.waitTextIntoEditor(EXPECTED_EDITOR_TEXT);
+    assertTrue(editor.getVisibleTextFromEditor().contains("package javax.servlet"));
```

Comment:
This tests for less text than before. Why not check the whole text?

Reply:
`waitTextIntoEditor` checks only visible text. So, it depends on the screen resolution.
In master branch it contains decomplied code which fits the screen. In 5730 branch it contains source code, that's why I've changed the expectation.